corehq/apps/sso/tests/test_tasks.py

```diff
@@ -84,6 +89,43 @@ def test_renew_service_provider_x509_certificates(self):
             new_cert_expiration.replace(tzinfo=None)
         )
 
+    def assert_idp_cert_expires_reminder(self, assert_reminder, remind_days=IDP_CERT_EXPIRES_REMINDER_DAYS):
+        # setup account attrs
+        recipient = "admin@example.com"
+        self.account.enterprise_admin_emails = [recipient]
+        self.account.save()
+        # configure idp expiration date
+        expires = datetime.datetime.utcnow()
+        self.idp.date_idp_cert_expiration = expires
+        self.idp.save()
+        # test alerts
+        print(f"the IdP cert is set to expire today ({expires})")
```

Comment:
best practice to not leave in print statements ^_^

Reply:
Ugh, how did that slip through? :facepalm: